Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ProcessedFile/ByFile.tsx

```diff
@@ -25,22 +25,40 @@ import * as React from 'react';
 import { Table, Column } from '@gpa-gemstone/react-table';
 import * as _ from 'lodash';
 import { Application, OpenXDA } from '@gpa-gemstone/application-typings';
-
 import { DefaultSearchField } from '../CommonComponents/SearchFields';
-import { SearchBar, Search, Modal, LoadingIcon, LoadingScreen } from '@gpa-gemstone/react-interactive';
+import { SearchBar, Search, Modal, LoadingIcon, LoadingScreen, ToolTip } from '@gpa-gemstone/react-interactive';
 import { useAppSelector, useAppDispatch } from '../hooks';
-import { DataFileSlice } from '../Store/Store';
+import { DataFileSlice, ConfigSlice } from '../Store/Store';
 import { OpenXDA as GlobalXDA } from '../global';
 import moment from 'moment';
 import { Paging } from '@gpa-gemstone/react-table';
-import { ReactIcons } from '@gpa-gemstone/gpa-symbols';
-
+import { CrossMark, ReactIcons } from '@gpa-gemstone/gpa-symbols';
```

Comment:
I thought `CrossMark` is depreciated

Reply:
It should be, I will get rid of it I don't recall why I added this instead of ReactIcons.